papas/mergedclusterbuilder.py

```diff
@@ -67,14 +67,18 @@ def __init__(self, clusters, ruler, history_nodes = None):
     def _make_merged_clusters(self):
         #carry out the merging of linked clusters
         for subgraphids in self.subgraphs:
-            subgraphids.sort()
+            subgraphids.sort(reverse=True) #start with biggest clusters 
```

Comment:
At present there is no (re)ordering on the size of the subgraphs/blocks. The sort is on the members of a subgraph from highest energy to lowest energy Cluster.